src/makeRobotAccountant.js

```diff
@@ -4,9 +4,19 @@
  *
  * @return {function}
  */
-
+//  few years ago I worked as accountant and it`s really  BORING
```

Comment:
It was in the past, and now you're giving robots a headache 🤖

Reply:
:harold: